huntsman/camera/pyro.py

```diff
@@ -575,20 +570,27 @@ def get_uid(self):
         """
         return self._camera.uid
 
-    def take_exposure(self, seconds, base_name, dark, *args, **kwargs):
-        # Using the /./ syntax for partial relative paths (needs rsync >= 2.6.7)
-        filename = os.path.join(os.path.abspath(self.config['directories']['images']),
-                                './',
-                                base_name)
+    def take_exposure(self, seconds, filename, dark, *args, **kwargs):
+        
+        #Make sure the filename is accessible on the SSHFS
+        basename = os.path.normpath(self.config['directories']['base'])
+        filename = os.path.normpath(filename)
+        if not filename.startswith(basename):
+            raise ValueError("Filename not accesible on SSHFS.")
+        
+        #Update the filename for the SSHFS
+        mountpoint = os.path.normpath(self.config['directories']['sshfs_mount'])
+        filename = os.path.join(mountpoint, filename)
+        
+                
         # Start the exposure and wait for it complete
         self._camera.take_exposure(seconds=seconds,
                                    filename=filename,
                                    dark=dark,
                                    blocking=True,
                                    *args,
                                    **kwargs)
-        # Return the user@host:/path for created file to enable it to be moved over the network.
-        return "{}@{}:{}".format(self.user, self.host, filename)
+        return None
```

Comment:
Should still return just the plain filename. 

```suggestion
        return filename
```